### PR TITLE
workbench:  updates

### PR DESCRIPTION
**Pull request by deepfire**
- allow profiling mode control via `WB_PROFILING_MODE` make variable -- which defaults to `time`, can be set to `space`
- analysis: eliminate logfile prefiltering
- add `forge-stress-large` profiles, with three times the useful data
- more analysis fixes

### CODE REVIEW

File: bench/locli/src/Cardano/Command.hs [hunk 1]
```diff
@@ -50,7 +50,7 @@ data ChainCommand
   |        ReadMetaGenesis  (JsonInputFile  RunPartial) (JsonInputFile  Genesis)
   |        WriteMetaGenesis TextOutputFile              TextOutputFile
 
-  |        Unlog            (JsonInputFile (RunLogs ())) Bool [LOAnyType]
+  |        Unlog            (JsonInputFile (RunLogs ())) Bool (Maybe [LOAnyType])
```
Comment (mgmeier) [inline bench/locli/src/Cardano/Command.hs:53]: ❤️

File: bench/locli/src/Cardano/Unlog/LogObject.hs [hunk 2]
```diff
@@ -99,11 +99,16 @@ readLogObjectStream f okDErr anyOks =
...
-    filter ((\case
+    filter ((case loAnyLimit of
+              Nothing -> \case
+                LOAny{} -> False
+                _       -> True
+              Just constraint -> \case
                 LOAny laty obj ->
-                  if elem laty anyOks then True else
+                  if elem laty constraint
+                  then True else
```
Comment (mgmeier) [inline bench/locli/src/Cardano/Unlog/LogObject.hs:109]: not just `elem laty anyOks || error (printf "...")` ? ;-)

**Review by mgmeier — APPROVED** [review]
Thank you, much needed.

File: bench/locli/src/Cardano/Unlog/LogObject.hs (hunk 2 above)
Comment (deepfire) [reply bench/locli/src/Cardano/Unlog/LogObject.hs:109]: The idea is that having an explicit `True` sort of rhymes with other case branches.

**Pull request merged**